publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/create/CreatePublishedPublicationHandler.java

```diff
@@ -68,7 +68,7 @@ private String randomUnitUser() {
     }
     
     private Publication storeAsPublishedPublication(Publication publication) {
-        return resourceService.createPublicationFromImportedEntry(publication);
+        return resourceService.createPublicationFromImportedEntry(publication, null);
```

Comment:
Is this handler in use?

Reply:
I don't think so. It seems like the handler, which managed the publication creation by scopus-import, which was located in the fetch-doi project when scopus-Import was using api calls to create the publications.

Can be removed.